CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Changelog for SendFS
 
 ## [Unreleased]
 
+## [0.6.2] - 2024-09-02
+
+### Fixed
+
+- Copy command in makefile for about page (#122)
```

Comment:
`makefile` is nothing for the end user. Better something like "About page link not working when served from NeoFS"